pageserver/src/tenant/storage_layer.rs

```diff
@@ -8,6 +8,8 @@ pub(crate) mod layer;
 mod layer_desc;
 mod layer_name;
 pub mod merge_iterator;
+use tokio::sync::{self};
+use utils::bin_ser::BeSer;
```

Comment:
nit: let's move this down to the use blocks below.

Reply:
Just that these `use` statements were in the middle of a `mod` block. Seems fixed now.